.github/workflows/cron.yml

```diff
@@ -0,0 +1,43 @@
+name: cron
+
+on:
+  schedule:
+    - cron: '0 5 * * 1' # Run every Monday at 6 AM (UTC+1) - Paris
+  workflow_dispatch: # To debug.
+
+concurrency:
+  group: ci-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+env:
+  GO_VERSION: "1.21" # https://go.dev/dl/
+
+jobs:
+  update-go-dependencies:
```

Comment:
have you looked into something like Dependabot? I'm not sure if `Go` is supported, but I think it could be more robust

Reply:
Yes, you should use dependabot for this see https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuring-dependabot-version-updates

Reply:
Yes, closing this PR in favor of #200.